microsoft/surface/default.nix

```diff
@@ -1,46 +1,10 @@
-{ lib, pkgs, ... }:
-
-let
-  inherit (lib) mkDefault;
-
-in {
-  imports = [
-    ./kernel
-    ./firmware/surface-go/ath10k
+{ ... }:
+
+{
+  assertions = [
+    {
+      assertion = false;
+      message = "Importing microsoft/surface/ (default.nix) directly is deprecated! See microsoft/surface/OLD-BEHAVIOUR-DEPRECATED.md for more details.";
```

Comment:
```suggestion
      message = "Importing microsoft/surface/ (default.nix) directly is deprecated! See https://github.com/NixOS/nixos-hardware/blob/master/microsoft/surface/OLD-BEHAVIOUR-DEPRECATED.md for more details.";
```

Reply:
This is easier to click/read in the terminal.

Reply:
> This is easier to click/read in the terminal.

That's an excellent idea, thanks!